lib/mm_network.pm

```diff
@@ -56,7 +56,7 @@ sub is_networkmanager {
 sub configure_static_ip {
     my (%args) = @_;
     my $ip = $args{ip};
-    my $mtu = $args{mtu} // 1458;
+    my $mtu = $args{mtu} // 1380;
```

Comment:
I feel like this should probably come from a config file? Since it's clearly dependant on the environment.

Reply:
yes, we can do that. However changing the default value still makes sense